app/api/services/informationextraction/getFiles.ts

```diff
@@ -16,13 +17,15 @@ interface FileWithAggregation {
   entity: string;
   language: string;
   extractedMetadata: ExtractedMetadataSchema[];
+  label_text: string;
```

Comment:
If this is the value from the entity, this is not necessarily just a string, right? For example, for dates and numbers it is a number.